src/api/licenses.js

```diff
@@ -42,4 +42,31 @@ const getAllLicenseApi = ({ page, limit, groupName, kind }) => {
   });
 };
 
-export default getAllLicenseApi;
+export const createCandidateLicenseApi = ({
+  shortName,
+  fullName,
+  text,
+  risk,
+  licenseUrl,
+  isCandidate,
+  mergeRequest,
+}) => {
+  const url = endpoints.license.createCandidateLicense();
+  return sendRequest({
+    url,
+    method: "POST",
+    headers: {
+      Authorization: getToken(),
+      groupName: "",
+    },
+    body: {
+      shortName,
+      fullName,
+      text,
+      risk,
+      url: licenseUrl,
+      isCandidate,
```

Comment:
In case this function is specific to candidate licenses, the `isCandidate` can be marked as true.